Makefile

```diff
@@ -0,0 +1,16 @@
+.MINO: agile
```

Comment:
Can you please add into README file how to use Makefile please?

Reply:
done!